contrib/sys.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env drgn
+# Copyright (c) SUSE Linux.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""Display system information and configuration data."""
+
+from datetime import datetime
+from datetime import timedelta
+
+from drgn.helpers.linux import for_each_online_cpu
+from drgn.helpers.linux.pid import for_each_task
+
+
+def print_line(key, value):
+    print(f"{key:<16} {value}")
+
+
+uts = prog["init_uts_ns"].name
+
+timekeeper = prog["shadow_timekeeper"]
+date = datetime.fromtimestamp(timekeeper.xtime_sec).strftime("%c")
+uptime = timedelta(seconds=timekeeper.ktime_sec.value_())
+avenrun = [f"{x.value_() / (1 << 11):.2f}" for x in prog["avenrun"]]
```

Comment:
This would make a great helper if you're interested in adding that. One complication is that some versions of GCC (including the one used to build the vmtest kernel) generate bad debug info for `avenrun` that doesn't include the array length:

```
$ python3 -m vmtest.vm -k '*' python3 -Bm drgn
...
>>> prog['avenrun']
(unsigned long []){}
```

So you might want to explicitly iterate with something like:

```python3
avenrun = prog["avenrun"]
loadavg = [f"{avenrun[i].value_() / (1 << 11):.2f}" for i in range(3)]
```

Reply:
Works for me, let's do it as #265.